src/Compilers/CSharp/Test/Symbol/Symbols/ModuleInitializersTests.Targets.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Symbols
+{
+    public sealed partial class ModuleInitializersTests
+    {
```

Comment:
>{ [](start = 4, length = 1)

It feels like it would be good to cover scenario when an attribute target doesn't allow placing the attribute on the method. For each scenario covered in this file.